modules/data.atmosphere/R/tdm_temporal_downscale.R

```diff
@@ -55,8 +55,10 @@ predict.subdaily <- function(dat.mod, n.ens, path.model, lags.list = NULL,
     # propogated
     dat.sim <- list()
     
-    # ----- Modeling surface_downwelling_shortwave_flux_in_air 
-    {
+    # ----- Modeling surface_downwelling_shortwave_flux_in_air
+    # The following bracket encloses the shortwave_flux downscaling section. The same 
+    # format is replicated for each variable. 
+    { 
```

Comment:
better comments are great, but I am still confused why the braces are used. 

If they have no effect they should be removed.

But the bigger issue is that they delineate chunks of code that are mostly repeated, and in which there appear to be a fairly limited amount of variable-specific code that could be handled with `if ... else `. It will be much easier to go ahead and write a function now than after the sections evolve apart from each other. And much easier to maintain in the long run. 

I know this is a major change, but I am so confident that it will be worthwhile I've drafted a first pass at this here: https://gist.github.com/dlebauer/15bc2d58310bc4e6d76882f61da40caf and can help if you need.